src/config/vaults.ts

```diff
@@ -0,0 +1,21 @@
+import { CurrencyIdLiteral } from '@interlay/interbtc-api';
+
```

Comment:
Wouldn't be more appropriate to use `CurrencySymbols` instead of `CurrencyIdLiteral` type in here?

Reply:
Yeah good point, this is a bit tricky - in some cases idLiterals and symbols are different (e.g. KBTC/kBTC). I'll have a think about how this can be approved when I add KINT support (which I'm doing today).